app/templates/components/component-tree-item.hbs

```diff
@@ -31,18 +33,18 @@
       }}
     </button>
 
-    {{#if hasElement}}
+    {{#if (eq @item.view.tagName "")}}
```

Comment:
Why the switch here?

Reply:
I did this so I could kill `component-tree-item.js`.

Reply:
That was my concern, that this might not work later, and it's not clear, to the casual observer what this check is for necessarily. I would prefer to keep something like `hasElement` and heavily document what we are checking for in the JS. That way, we can keep things from getting murky as we rewrite things. I want it to all be very well documented 😃

Reply:
I created a helper, is that cool?

Reply:
A helper sounds good 👍